styles/books/generic/webview.scss

```diff
@@ -125,7 +125,7 @@
 //Exercises
 @include add_settings((
     Exercise: (
-        _selectors: ('[data-type="example"]', '.example', '[data-type="exercise"]', '.exercise'),
```

Comment:
removed selectors that never appear in the content, or (in the case of `[data-type=example]` as the root exercise selector) are already covered (by `[data-type=exercise]`)